spec/game_spec.rb

```diff
@@ -1,24 +1,32 @@
 require_relative '../game'
+require_relative '../game_view'
+require_relative '../validator'
 require_relative 'mocks/mock_io_handler'
 
 describe 'Game' do
 
+  game_view = GameView.new
   mock_io_handler = MockIOHandler.new
   prompter = Prompter.new
-  game = Game.new(mock_io_handler, prompter)
+  validator = Validator.new
+  game = Game.new(game_view, mock_io_handler, prompter, validator)
 
-  describe 'game_handler' do
+  describe '#game_handler' do
```

Comment:
This is a good test for the most part, but it is limited in that it does not actually test the main functionality of the game. You are testing the side effects, which is good, but this does not prove that you can play a game of hangman.

When we have game ending conditions in place, we will have to revisit these tests.